server/statistics/metrics.go

```diff
@@ -71,6 +71,54 @@ var (
 			Name:      "label_level",
 			Help:      "Number of regions in the different label level.",
 		}, []string{"type"})
+	readByteHist = prometheus.NewHistogram(
```

Comment:
HotReadBytesHist? It only observes the hot peer

Reply:
All region which heartbeat will be observed